helm/default-apps-aws/values.yaml

```diff
@@ -202,6 +202,26 @@ apps:
     # used by renovate
     # repo: giantswarm/observability-bundle
     version: 0.7.2
+  teleport-kube-agent:
+    appName: teleport-kube-agent
+    chartName: teleport-kube-agent
+    catalog: default-test
+    clusterValues:
+      configMap: true
+      secret: true
+    enabled: true
+    forceUpgrade: true
+    namespace: kube-system
+    # used by renovate
+    # repo: giantswarm/teleport-kube-agent-app
+    version: 0.4.0-f06a6d1a33515030b2c7cbd98d10d9f5bdcbe2cb
```

Comment:
All of this looks good to me but let's only merge it after testing thorougly and releasing teleport :)